invenio_rdm_records/services/pids/providers/datacite.py

```diff
@@ -209,3 +209,23 @@ def validate(self, record, identifier=None, provider=None, **kwargs):
             errors.append(str(e))
 
         return (True, []) if not errors else (False, errors)
+
+    def validate_record(self, record):
+        """Validate the record according to DataCite rules.
+
+        We only add check for values not already covered by the record schema.
+
+        :returns: A tuple (success, errors). `success` is a bool that specifies
+                  if the validation was successful. `errors` is an
+                  error dict of the form: `{"<fieldA>": ["<msgA1>", ...], ...}`.
+        """
+        errors = {}
+
+        if not record["metadata"].get("publisher"):
+            errors.update(
+                {
+                    "metadata.publisher": [_("Missing data for required field.")],
```

Comment:
Maybe in this case we could go with an even more specific error message indicating exactly why this field is required? 
```suggestion
                    "metadata.publisher": [_("The publisher field is required for doi registration.")],
```

Reply:
maybe... would have rather foregone the need for new text for translators to translate :smile:

Reply:
Done :)